src/lib/journey/stages/generic.svelte

```diff
@@ -68,12 +69,13 @@
     }
   });
 
+  onMount(() => captureLinks(linkWrapper, journey));
```

Comment:
This ensures the link wrapper and the capture links function bind after the component mounts. Since these links are added via HTML strings from an external source, we can't handle them natively in Svelte. So, we have to handle this is native JS.